src/module/skb/skb.rs

```diff
@@ -1,17 +1,30 @@
-use anyhow::Result;
+use std::mem;
+
+use anyhow::{bail, Result};
 use clap::Args;
 
-use super::skb_hook;
+use super::{bpf::*, skb_hook};
 use crate::{
     cli::{dynamic::DynamicCommand, CliConfig},
     collect::Collector,
-    core::{events::bpf::BpfEvents, probe::{ProbeManager, Hook}},
+    core::{
+        events::bpf::{BpfEventOwner, BpfEvents},
+        probe::{Hook, ProbeManager},
+    },
 };
 
 const SKB_COLLECTOR: &str = "skb";
 
-#[derive(Args)]
+#[derive(Args, Default)]
 pub(crate) struct SkbCollectorArgs {
+    #[arg(
+        long,
+        help = "Comma separated list of data to collect from SKBs.
+
+Possible values: all, l2.
+Default value:"
+    )]
+    skb_sections: Option<String>,
```

Comment:
clap supports arguments with type `Vec<String>` which can hold multiple values and support default values and "possible values" (the `--collectors` is done this way).

I believe this could be implemented this way making it much simpler since clap would autogenerate the help.

I tried a little experiment to confirm it works with the Derive interface:

```rust
use clap::{arg, builder::PossibleValuesParser, Parser};

#[derive(Parser)]
#[command(about = "I am a program", long_about = None)]
struct Cli {
    #[arg(short,
          long,
          value_parser=PossibleValuesParser::new(["foo", "bar", "baz", "all"]),
          value_delimiter=',',
          default_value="foo,bar")]
    opt: Vec<String>,
}

fn main() {
    let cli = Cli::parse();
    println!("{:?}", cli.opt);
}
```
```shell
> ./target/debug/clappossible -h
I am a program and I work, just pass `-h`

Usage: clappossible [OPTIONS]

Options:
  -o, --opt <OPT>  [default: foo,bar] [possible values: foo, bar, baz, all]
  -h, --help       Print help

> ./target/debug/clappossible 
["foo", "bar"]
> /target/debug/clappossible --opt bar,baz
["bar", "baz"]
```